PEP8 cleanup

Hello,

This code need some PEP8 cleanup

```
$ flake8 *.py
apc.py:22:17: E221 multiple spaces before operator
apc.py:23:18: E221 multiple spaces before operator
apc.py:37:1: E302 expected 2 blank lines, found 1
apc.py:48:21: E999 SyntaxError: invalid syntax
apc.py:95:28: W602 deprecated form of raising exception
apc.py:100:80: E501 line too long (81 > 79 characters)
apc.py:191:9: E265 block comment should start with '# '
apc.py:204:1: E302 expected 2 blank lines, found 1
apc.py:206:41: E251 unexpected spaces around keyword / parameter equals
apc.py:206:43: E251 unexpected spaces around keyword / parameter equals
apc.py:206:60: E251 unexpected spaces around keyword / parameter equals
apc.py:206:62: E251 unexpected spaces around keyword / parameter equals
apc.py:207:9: E128 continuation line under-indented for visual indent
apc.py:207:13: E251 unexpected spaces around keyword / parameter equals
apc.py:207:15: E251 unexpected spaces around keyword / parameter equals
apc.py:209:9: E128 continuation line under-indented for visual indent
apc.py:210:42: E251 unexpected spaces around keyword / parameter equals
apc.py:210:44: E251 unexpected spaces around keyword / parameter equals
apc.py:211:9: E128 continuation line under-indented for visual indent
apc.py:211:13: E251 unexpected spaces around keyword / parameter equals
apc.py:211:15: E251 unexpected spaces around keyword / parameter equals
apc.py:212:41: E251 unexpected spaces around keyword / parameter equals
apc.py:212:43: E251 unexpected spaces around keyword / parameter equals
apc.py:212:60: E251 unexpected spaces around keyword / parameter equals
apc.py:212:62: E251 unexpected spaces around keyword / parameter equals
apc.py:213:9: E128 continuation line under-indented for visual indent
apc.py:213:13: E251 unexpected spaces around keyword / parameter equals
apc.py:213:15: E251 unexpected spaces around keyword / parameter equals
apc.py:214:45: E251 unexpected spaces around keyword / parameter equals
apc.py:214:47: E251 unexpected spaces around keyword / parameter equals
apc.py:214:64: E251 unexpected spaces around keyword / parameter equals
apc.py:214:66: E251 unexpected spaces around keyword / parameter equals
apc.py:214:80: E501 line too long (87 > 79 characters)
apc.py:215:9: E128 continuation line under-indented for visual indent
apc.py:215:13: E251 unexpected spaces around keyword / parameter equals
apc.py:215:15: E251 unexpected spaces around keyword / parameter equals
apc.py:216:42: E251 unexpected spaces around keyword / parameter equals
apc.py:216:44: E251 unexpected spaces around keyword / parameter equals
apc.py:217:9: E128 continuation line under-indented for visual indent
apc.py:217:13: E251 unexpected spaces around keyword / parameter equals
apc.py:217:15: E251 unexpected spaces around keyword / parameter equals
apc.py:218:43: E251 unexpected spaces around keyword / parameter equals
apc.py:218:45: E251 unexpected spaces around keyword / parameter equals
apc.py:219:9: E128 continuation line under-indented for visual indent
apc.py:219:13: E251 unexpected spaces around keyword / parameter equals
apc.py:219:15: E251 unexpected spaces around keyword / parameter equals
apc.py:220:40: E251 unexpected spaces around keyword / parameter equals
apc.py:220:42: E251 unexpected spaces around keyword / parameter equals
apc.py:221:9: E128 continuation line under-indented for visual indent
apc.py:221:13: E251 unexpected spaces around keyword / parameter equals
apc.py:221:15: E251 unexpected spaces around keyword / parameter equals
apc.py:222:39: E251 unexpected spaces around keyword / parameter equals
apc.py:222:41: E251 unexpected spaces around keyword / parameter equals
apc.py:223:9: E128 continuation line under-indented for visual indent
apc.py:223:13: E251 unexpected spaces around keyword / parameter equals
apc.py:223:15: E251 unexpected spaces around keyword / parameter equals
apc.py:236:25: W602 deprecated form of raising exception
apc.py:249:29: W602 deprecated form of raising exception
apc.py:253:1: E305 expected 2 blank lines after class or function definition, found 1
lockfile.py:12:13: E401 multiple imports on one line
lockfile.py:16:1: E302 expected 2 blank lines, found 1
lockfile.py:19:1: E305 expected 2 blank lines after class or function definition, found 1
lockfile.py:19:1: E402 module level import not at top of file
lockfile.py:41:36: E999 SyntaxError: invalid syntax
lockfile.py:55:46: E231 missing whitespace after ','
lockfile.py:57:29: E231 missing whitespace after ','
lockfile.py:70:47: E231 missing whitespace after ','
lockfile.py:86:1: E303 too many blank lines (3)
lockfile.py:112:5: E303 too many blank lines (2)
lockfile.py:155:80: E501 line too long (81 > 79 characters)
lockfile.py:176:5: E303 too many blank lines (2)
lockfile.py:187:80: E501 line too long (80 > 79 characters)
lockfile.py:212:1: W391 blank line at end of file
```

Kind regards

--- a/apc.py
+++ b/apc.py
@@ -9,7 +9,6 @@ Tested with AP7900, but should work with similar models.
 '''
 
 import pexpect
-import sys
 import re
 import time
 
@@ -19,8 +18,8 @@ from lockfile import FilesystemLock
 APC_ESCAPE = '\033'
 
 APC_IMMEDIATE_REBOOT = ['4', '3']
-APC_IMMEDIATE_ON     = ['1', '1']
-APC_IMMEDIATE_OFF    = ['3', '2']
+APC_IMMEDIATE_ON = ['1', '1']
+APC_IMMEDIATE_OFF = ['3', '2']
 
 APC_YES = 'YES'
 APC_LOGOUT = '4'
@@ -34,6 +33,7 @@ APC_DEFAULT_HOST = '192.168.1.2'
 LOCK_PATH = '/tmp/apc.lock'
 LOCK_TIMEOUT = 60
 
+
 class APC:
     def __init__(self, options):
         self.host = options.host
@@ -45,15 +45,15 @@ class APC:
 
     def info(self, msg):
         if not self.quiet:
-            print msg
+            print(msg)
 
     def notify(self, outlet_name, state):
-        print 'APC %s: %s %s' % (self.host, outlet_name, state)
+        print('APC %s: %s %s' % (self.host, outlet_name, state))
 
     def sendnl(self, a):
         self.child.send(a + '\r\n')
         if self.verbose:
-            print self.child.before
+            print(self.child.before)
 
     def _lock(self):
         self.info('Acquiring lock %s' % (LOCK_PATH))
@@ -92,12 +92,13 @@ class APC:
         match = APC_VERSION_PATTERN.search(header)
 
         if not match:
-            raise Exception, 'Could not parse APC version'
+            raise Exception('Could not parse APC version')
 
         self.version = match.group(1)
         self.is_new_version = (self.version[0] == '3')
 
-        self.info('Logged in as user %s, version %s' % (self.user, self.version))
+        self.info('Logged in as user %s, version %s'
+                  % (self.user, self.version))
 
     def get_outlet(self, outlet):
         if str(outlet) in ['*', '+', '9']:
@@ -188,39 +189,41 @@ class APC:
         self.child.interact()
 
     def disconnect(self):
-        #self._escape_to_main()
+        # self._escape_to_main()
 
         self.sendnl(APC_LOGOUT)
         self.child.sendeof()
         if not self.quiet:
-            print 'DISCONNECTED from %s' % self.host
+            print('DISCONNECTED from %s' % self.host)
 
         if self.verbose:
-            print '[%s]' % ''.join(self.child.readlines())
+            print('[%s]' % ''.join(self.child.readlines()))
 
         self.child.close()
         self._unlock()
 
+
 def main():
     parser = ArgumentParser(description='APC Python CLI')
-    parser.add_argument('--host', action = 'store', default = APC_DEFAULT_HOST,
-        help = 'Override the host')
+    parser.add_argument('--host', action='store', default=APC_DEFAULT_HOST,
+                        help='Override the host')
     parser.add_argument('-v', '--verbose', action='store_true',
-        help='Verbose messages')
-    parser.add_argument('--quiet', action = 'store_true',
-        help = 'Quiet')
-    parser.add_argument('--user', action = 'store', default = APC_DEFAULT_USER,
-        help = 'Override the username')
-    parser.add_argument('--password', action = 'store', default = APC_DEFAULT_PASSWORD,
-        help = 'Override the password')
-    parser.add_argument('--debug', action = 'store_true',
-        help = 'Debug mode')
-    parser.add_argument('--reboot', action = 'store',
-        help = 'Reboot an outlet')
-    parser.add_argument('--off', action = 'store',
-        help = 'Turn off an outlet')
-    parser.add_argument('--on', action = 'store',
-        help = 'Turn on an outlet')
+                        help='Verbose messages')
+    parser.add_argument('--quiet', action='store_true',
+                        help='Quiet')
+    parser.add_argument('--user', action='store', default=APC_DEFAULT_USER,
+                        help='Override the username')
+    parser.add_argument('--password', action='store',
+                        default=APC_DEFAULT_PASSWORD,
+                        help='Override the password')
+    parser.add_argument('--debug', action='store_true',
+                        help='Debug mode')
+    parser.add_argument('--reboot', action='store',
+                        help='Reboot an outlet')
+    parser.add_argument('--off', action='store',
+                        help='Turn off an outlet')
+    parser.add_argument('--on', action='store',
+                        help='Turn on an outlet')
 
     args = parser.parse_args()
 
@@ -232,8 +235,8 @@ def main():
 
     try:
         apc = APC(args)
-    except pexpect.TIMEOUT, e:
-        raise SystemExit, 'ERROR: Timeout connecting to APC'
+    except pexpect.TIMEOUT as e:
+        raise SystemExit('ERROR: Timeout connecting to APC')
 
     if args.debug:
         apc.debug()
@@ -245,10 +248,11 @@ def main():
                 apc.on(args.on)
             elif args.off:
                 apc.off(args.off)
-        except pexpect.TIMEOUT, e:
-            raise SystemExit, 'APC failed!  Pexpect result:\n%s' % e
+        except pexpect.TIMEOUT as e:
+            raise SystemExit('APC failed!  Pexpect result:\n%s' % e)
         finally:
             apc.disconnect()
+
 
 if __name__ == '__main__':
     main()

--- a/lockfile.py
+++ b/lockfile.py
@@ -9,12 +9,10 @@ Filesystem-based interprocess mutex.
 
 __metaclass__ = type
 
-import errno, os
+import errno
+import os
 
 from time import time as _uniquefloat
-
-def unique():
-    return str(long(_uniquefloat() * 1000))
 
 from os import rename
 try:
@@ -38,7 +36,7 @@ except:
         def kill(pid, signal):
             try:
                 OpenProcess(0, 0, pid)
-            except pywintypes.error, e:
+            except pywintypes.error as e:
                 if e.args[0] == ERROR_ACCESS_DENIED:
                     return
                 elif e.args[0] == ERROR_INVALID_PARAMETER:
@@ -47,14 +45,17 @@ except:
             else:
                 raise RuntimeError("OpenProcess is required to fail.")
 
-    _open = file
+    _open = file  # noqa
+
+    def unique():
+        return str(long(_uniquefloat() * 1000))  # noqa
 
     # XXX Implement an atomic thingamajig for win32
     def symlink(value, filename):
         newlinkname = filename+"."+unique()+'.newlink'
-        newvalname = os.path.join(newlinkname,"symlink")
+        newvalname = os.path.join(newlinkname, "symlink")
         os.mkdir(newlinkname)
-        f = _open(newvalname,'wcb')
+        f = _open(newvalname, 'wcb')
         f.write(value)
         f.flush()
         f.close()
@@ -67,8 +68,8 @@ except:
 
     def readlink(filename):
         try:
-            fObj = _open(os.path.join(filename,'symlink'), 'rb')
-        except IOError, e:
+            fObj = _open(os.path.join(filename, 'symlink'), 'rb')
+        except IOError as e:
             if e.errno == errno.ENOENT or e.errno == errno.EIO:
                 raise OSError(e.errno, None)
             raise
@@ -80,7 +81,6 @@ except:
     def rmlink(filename):
         os.remove(os.path.join(filename, 'symlink'))
         os.rmdir(filename)
-
 
 
 class FilesystemLock:
@@ -108,7 +108,6 @@ class FilesystemLock:
     def __init__(self, name):
         self.name = name
 
-
     def lock(self):
         """
         Acquire this lock.
@@ -123,7 +122,7 @@ class FilesystemLock:
         while True:
             try:
                 symlink(str(os.getpid()), self.name)
-            except OSError, e:
+            except OSError as e:
                 if _windows and e.errno in (errno.EACCES, errno.EIO):
                     # The lock is in the middle of being deleted because we're
                     # on Windows where lock removal isn't atomic.  Give up, we
@@ -132,13 +131,13 @@ class FilesystemLock:
                 if e.errno == errno.EEXIST:
                     try:
                         pid = readlink(self.name)
-                    except OSError, e:
+                    except OSError as e:
                         if e.errno == errno.ENOENT:
                             # The lock has vanished, try to claim it in the
                             # next iteration through the loop.
                             continue
                         raise
-                    except IOError, e:
+                    except IOError as e:
                         if _windows and e.errno == errno.EACCES:
                             # The lock is in the middle of being
                             # deleted because we're on Windows where
@@ -150,13 +149,14 @@ class FilesystemLock:
                     try:
                         if kill is not None:
                             kill(int(pid), 0)
-                    except OSError, e:
+                    except OSError as e:
                         if e.errno == errno.ESRCH:
-                            # The owner has vanished, try to claim it in the next
+                            # The owner has vanished,
+                            # try to claim it in the next
                             # iteration through the loop.
                             try:
                                 rmlink(self.name)
-                            except OSError, e:
+                            except OSError as e:
                                 if e.errno == errno.ENOENT:
                                     # Another process cleaned up the lock.
                                     # Race them to acquire it in the next
@@ -172,7 +172,6 @@ class FilesystemLock:
             self.clean = clean
             return True
 
-
     def unlock(self):
         """
         Release this lock.
@@ -184,7 +183,7 @@ class FilesystemLock:
         """
         pid = readlink(self.name)
         if int(pid) != os.getpid():
-            raise ValueError("Lock %r not owned by this process" % (self.name,))
+            raise ValueError("Lock %r not owned by this process" % self.name)
         rmlink(self.name)
         self.locked = False
 
@@ -209,4 +208,3 @@ def isLocked(name):
 
 
 __all__ = ['FilesystemLock', 'isLocked']
-

